<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="pickFormat">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>270</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Choose here</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>mp4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>wmv</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>avi</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>250</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose desired format</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_convert">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>350</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_openfile">
    <property name="geometry">
     <rect>
      <x>231</x>
      <y>221</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Open File Explorer</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_filename">
    <property name="geometry">
     <rect>
      <x>369</x>
      <y>221</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>No file chosen</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_error">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>440</y>
      <width>701</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
background-color: rgb(236, 96, 96);
border-style: outset;
border-width: 1 px;
border-color: red
</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_path">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>310</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose desired path</string>
    </property>
   </widget>
   <widget class="QLabel" name="l_opath">
    <property name="geometry">
     <rect>
      <x>308</x>
      <y>310</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>No path chosen</string>
    </property>
   </widget>
   <widget class="QLabel" name="progress">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>390</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>In progress... Please, wait</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
